include/open62541/server_pubsub.h

```diff
@@ -552,7 +552,7 @@ UA_EXPORT UA_StatusCode UA_THREADSAFE
 UA_Server_unfreezeWriterGroupConfiguration(UA_Server *server, const UA_NodeId writerGroup);
 
 UA_EXPORT UA_StatusCode UA_THREADSAFE
-UA_Server_setWriterGroupOperational(UA_Server *server, const UA_NodeId writerGroup);
```

Comment:
Don’t break existing code.
Please define the old UA_Server_setWriterGroupOperational as a macro alias for the new API.
You can add the UA_DEPRECATED attribute.
So developers get a compiler warning, but the code still works.
We can then remove the old API for the over ext stable release.

Reply:
Okay will do that